src/server/unit_sao.cpp

```diff
@@ -75,20 +75,22 @@ void UnitSAO::setAnimationSpeed(float frame_speed)
 	m_animation_speed_sent = false;
 }
 
-void UnitSAO::setBonePosition(const std::string &bone, v3f position, v3f rotation)
+void UnitSAO::setBoneOverride(const std::string &bone, BonePositionOverride *override)
 {
 	// store these so they can be updated to clients
-	m_bone_position[bone] = core::vector2d<v3f>(position, rotation);
-	m_bone_position_sent = false;
+	m_bone_override[bone] = override;
```

Comment:
Is this a memleak - should I delete the previous override?

Reply:
If the previous override won't be used again then yes.

Reply:
If it's kept anywhere else in the code set it to nullptr.